### DISCUSSION TITLE
v3.2.23

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#11208] [General]
## What's Changed
* add missing fa for import/export/components by @mszabeh in https://github.com/filamentphp/filament/pull/11196
* RenderHook constants by @margarizaldi in https://github.com/filamentphp/filament/pull/10996
* Set table column width by @divdax in https://github.com/filamentphp/filament/pull/11148
* Allow relationship filters using custom keys by @Skullbock in https://github.com/filamentphp/filament/pull/11164
* Allow disabling of ->defaultSort() by @ctoma in https://github.com/filamentphp/filament/pull/11096

## New Contributors
* @divdax made their first contribution in https://github.com/filamentphp/filament/pull/11148
* @ctoma made their first contribution in https://github.com/filamentphp/filament/pull/11096

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.22...v3.2.23

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.23'>v3.2.23</a>.</em>

**Comment by LucasJohnNyamhanga** [comment]
Excelling with brilliance, you've performed exceptionally well in this task. Outstanding job! 🌟
